cmake/compilers/MSVC.cmake

```diff
@@ -35,6 +35,13 @@ endif()
 set(TBB_LIB_COMPILE_FLAGS -D_CRT_SECURE_NO_WARNINGS /GS)
 set(TBB_COMMON_COMPILE_FLAGS /volatile:iso /FS /EHsc)
 
+set(TBB_LIB_LINK_FLAGS ${TBB_LIB_LINK_FLAGS} /DYNAMICBASE /GUARD:CF /NXCOMPAT)
```

Comment:
Is `/GUARD:CF` still needed?

Reply:
/GUARD:CF is only an optional flag as per SDL guidelines. It might have some performance overhead similar to Qcf-protection. Decided to remove this option.